src/pages/Quote.tsx

```diff
@@ -129,6 +126,10 @@ const Quote = () => {
 
   return (
     <div className="flex flex-col gap-x-6 px-4 py-6 lg:grid lg:grid-cols-5 lg:px-9 lg:py-8">
+      <Helmet>
```

Comment:
Bhai helmet also has a title prop. You can set title through the title prop.

Reply:
@alaminShaheen - I couldn't find any mention about the prop in the documentation.

https://github.com/staylor/react-helmet-async#readme

Reply:
It's not properly documented bhai, but its there. Typescript should show the props available.

Reply:
@alaminShaheen  - You were right. Find prop definitions. Adjusted code

![image](https://github.com/alaminShaheen/irc-ui/assets/124192994/e3cc828f-bcbd-4fbf-ad46-8c8d20a9c9c4)